src/winforms/toga_winforms/widgets/table.py

```diff
@@ -122,6 +122,10 @@ def get_selection(self):
     def set_on_select(self, handler):
         pass
 
+    def set_font(self, font):
+        if font:
+            self.native.Font = font.bind(self.interface.factory).native
```

Comment:
As with Android - will this propagate to the table after the table has been created? I have a little more confidence that this one will, but it probably needs a test of some kind.

Reply:
Here, the change **is** propagated correctly